ci/images/debian/Dockerfile.dotnet10

```diff
@@ -20,6 +21,13 @@ ENV PATH=${PATH}:/usr/local/bin:/root/.local/bin:${NVM_DIR}/versions/node/v${NOD
 COPY ci/images/debian/install.sh /tmp/
 
 RUN set -e; \
+    if [ -f /etc/apt/sources.list ]; then \
+      sed -i "s|http://deb.debian.org|$DEBIAN_REPO|g" /etc/apt/sources.list; \
+      sed -i "s|http://security.debian.org|$DEBIAN_REPO|g" /etc/apt/sources.list; \
```

Comment:
Can we have `DEBIAN_SECURITY_REPO` variable which could be the same nexus url. This will help reverting the hostname cleanly.